libraries/SITL/SIM_RF_RDS02UF.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+/*
+  Simulator for the RDS02UF rangefinder
+*/
+
+#include "SIM_RF_RDS02UF.h"
+
+#include <stdio.h>
+
+using namespace SITL;
+
+
+    const uint8_t crc8_table[256] = {
```

Comment:
Bitwise operations eliminate the need for tables.
This saves 256 bytes of flash memory.